src/main/java/goodgenerator/blocks/tileEntity/base/GT_MetaTileEntity_LargeTurbineBase.java

```diff
@@ -216,6 +216,17 @@ public boolean explodesOnComponentBreak(ItemStack aStack) {
         return true;
     }
 
+    public long getMaximumOutput() {
+        long aTotal = 0;
+        for (GT_MetaTileEntity_Hatch_Dynamo aDynamo : mDynamoHatches) {
```

Comment:
Imo if you use a for  you should either break after the first calculation of aTotal or sum aTotal across hatch hatches.

Reply:
now it is only calculated when structure changed

Reply:
That's good but this issue remains. For example here: https://github.com/GTNewHorizons/GoodGenerator/blob/2457ec629f260ee9db4f90258110e2035e0b2248/src/main/java/goodgenerator/blocks/tileEntity/base/GT_MetaTileEntity_LargeTurbineBase.java#L272
you sum voltage and amperage across all hatches, because its in a for-loop. Imo the same thing should be done here so its clean, even though practically the list only has one member.